src/constraints/constraints.jl

```diff
@@ -0,0 +1,232 @@
+export
+    DistanceConstraint,
+    check_position_constraints,
```

Comment:
Add docstrings to the exported functions.

Reply:
It's used by the `check_position_constraint` (and velo) functions. It's not explicitly needed right now since we only have one algorithm but I don't think its hurting anything?